src/app/components/applications/SSV/MyAccount/common/NewRemainingDays/NewRemainingDays.tsx

```diff
@@ -24,7 +24,7 @@ const NewRemainingDays = ({ cluster, withdrawState, isInputFilled = null }: Prop
   let warningLiquidationState: boolean;
   const clusterRunWay = cluster.newRunWay ?? cluster.runWay;
   let remainingDays: number = clusterRunWay;
-  const typeOfiIsInputFilled =  typeof isInputFilled === 'boolean';
+  const typeOfiIsInputFilled = typeof isInputFilled === 'boolean';
```

Comment:
this is redundant
use !!isInputFilled instead for if statement

Reply:
+